runtime/gc_realtime/Metronome.hpp

```diff
@@ -25,18 +24,15 @@
 #define METRONOME_HPP_
 
 /* @ddr_namespace: default */
-#include "j9.h"
-#include "j9cp.h"
-#include "j9cfg.h"
-#include "modronopt.h"
-#include "modronbase.h"
+#include "omr.h"
+#include "omrcfg.h"
 
 #include <assert.h>
 #define ASSERT_LEVEL 0
 #define assert1(expr) assert((ASSERT_LEVEL < 1) || (expr));
 #define assert2(expr) assert((ASSERT_LEVEL < 2) || (expr));
 
-#define MAX_UINT ((UDATA) (-1))
+#define MAX_UINT ((uintptr_t) (-1))
```

Comment:
This definition exists in OMR code already in `AllocationContextSegregated.cpp`
Also OMR has `UDATA_MAX` definition for this used widely.
Also there are multiple places in the code with `((uintptr_t) (-1))` and `((UDATA) (-1))`
@charliegracie I wondering may be it is time to do cleanup and use single proper definition everywhere (done by two separate PRs - one for OMR another for OpenJ9) ?